app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
- [ ] **[OPTIONAL]** I receive the link to reset my password and I can reset it just fine, but I strongly recommend you look into this. After requesting the link for password reset I get this error page. I would recommend you fix it by adding `data: { turbo: false }` to this line.

  ```
  <%= f.submit "Send me reset password instructions", data: { turbo: false } %>
  ```

  ![fjff](https://user-images.githubusercontent.com/81468675/209308801-93f442fa-7101-49f9-a408-929e5cbb2665.jpg)

Reply:
@julie-ify Thanks for the review and suggestion